moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -495,27 +495,25 @@ def go(self, joints=None, wait=True):
 
         elif joints is not None:
             try:
-                self.set_joint_value_target(self.get_remembered_joint_values()[joints])
-            except TypeError:
-                self.set_joint_value_target(joints)
-        if wait:
-            return self._g.move()
-        else:
-            return self._g.async_move()
-
-    def plan(self, joints=None):
-        """ Return a tuple of the motion planning results such as
-            (success flag : boolean, trajectory message : RobotTrajectory,
-             planning time : float, error code : MoveitErrorCodes) """
-        if type(joints) is JointState:
-            self.set_joint_value_target(joints)
-
-        elif type(joints) is Pose:
-            self.set_pose_target(joints)
-
-        elif joints is not None:
-            try:
-                self.set_joint_value_target(self.get_remembered_joint_values()[joints])
+                self
+  .set_joint_value_target(self.get_remembered_joint_values()[joints]) except TypeError
+      : self.set_joint_value_target(joints) if wait : return self._g.move() else : return self._g
+                                                                                       .async_move()
+
+                                                                                           def plan(self, joints = None)
+    : ""
+      " Return a tuple of the motion planning results such as
```

Comment:
Did the triple quotes `"""` get split onto two lines here? That breaks the formatting - python has a special meaning for triple quotes.